en/docs/design/api-security/api-authentication/disable-security.md

```diff
@@ -0,0 +1,19 @@
+# Disabling Security for APIs
+
+## Disabling security for a single API resource
+
+1. Click **Resources** to navigate to the Resources page in the Publisher.
```

Comment:
Let's add where Resources is exactly so they know where to click.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/5555/commits/fef17550bbb360bbf5e86e06bf21b021bcca9ff8